packages/node-sdk/src/types.ts

```diff
@@ -94,6 +94,11 @@ export interface Feature {
    */
   isEnabled: boolean;
 
+  /**
+   * Optional user-defined configuration if the feature is enabled.
+   */
+  config: any;
+
```

Comment:
again: looks like nodeSDK changes are not complete. Should we do this in a separate PR?

Reply:
It's a separate PR indeed